cmd/unifi_exporter/main.go

```diff
@@ -9,49 +9,70 @@ import (
 	"net/http"
 	"strings"
 	"time"
+	"io/ioutil"
 
+	"gopkg.in/yaml.v2"
 	"github.com/mdlayher/unifi"
 	"github.com/mdlayher/unifi_exporter"
 	"github.com/prometheus/client_golang/prometheus"
 )
 
+type Config struct {
+	TelemetryAddr string `yaml: "telemetryaddr"`
+	MetricsPath string `yaml: "metricspath"`
+	UnifiAddr string `yaml: "unifiaddr"`
+	Username string `yaml: "username"`
+	Password string `yaml: "password"`
+	Site string `yaml: "site"`
+	Insecure bool `yaml: "insecure"`
+	Timeout int `yaml: "timeout"`
```

Comment:
Maybe timeout should have the unit of measurement in its name, e.g. `timeout_secs`.